cmd/sonobuoy/app/results.go

```diff
@@ -143,6 +151,15 @@ func result(input resultsInput) error {
 			lastErr = err
 		}
 
+		//bypass if this plugin is called clusterHealthSummaryPluginName
+		if input.plugin == clusterHealthSummaryPluginName {
```

Comment:
I wouldn't put this here; there is a validation section where we make sure we don't have duplicates or reserved names being used. Its in `gen.go` (the client one, not the cmd one). `checkPluginNames` is the function name.

Technically someone could do sonobuoy gen, save the output and modify it to avoid this check then run it. I dont think its worth the time to really ensure people dont shoot themselves in the foot manually though. If we were that concerned we could just run that check here too rather than distributing this logic in multiple places.